src/main/scala/no/ndla/articleapi/model/Language.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Part of NDLA article_api.
+ * Copyright (C) 2016 NDLA
+ *
+ * See LICENSE
+ *
+ */
+
+package no.ndla.articleapi.model
```

Comment:
Mapping-tjenesten gir oss en mapping fra ISO639-2 til ISO639-1. Dette objektet brukes for å angi hvilke språk som er støttet i søk (og i språkanalysene). Vi kan ikke dynamisk finne legge til/fjerne støttede språk i søk. Derfor trenger vi denne.